lua/sqlite/defs.lua

```diff
@@ -19,8 +19,11 @@ local clib = (function()
   end
 
   local clib_path = path
-    or (function() --- try to find libsqlite. Macos support only.
+    or (function() --- try to find libsqlite.Linux and Macos support only.
       local os = luv.os_uname()
+      if os.sysname == "Linux" then
```

Comment:
What if the architecture is different? e.g. arm. Is this is unified across all linux distros?

Reply:
> What if the architecture is different? e.g. arm. Is this is unified across all linux distros?

Thank you for your input.
I have now implemented a path checker in the code and it now works on (Ubuntu,Fedora,Arch) tested in a docker container
I am not completely sure about arm64 (as I do not have one) but I assume it should work there as well as I tried to cover all possible path for libsqlite3.so

Reply:
That sufficient, users can always set a variable to where the clib is. 

Thanks @Mine-Ranger